lib/matplotlib/animation.py

```diff
@@ -1776,3 +1786,11 @@ def _draw_frame(self, framedata):
                 a.set_animated(self._blit)
 
     save_count = _api.deprecate_privatize_attribute("3.7")
+
+
+def _validate_grabframe_kwargs(savefig_kwargs):
+    for k in {'dpi', 'bbox_inches', 'format'}:
```

Comment:
Make this a list so that the error message is deterministic when multiple invalid arguments are passed.

Reply:
(Or better a tuple which is not slower than a list...)